### ISSUE TITLE
Edit ACL tags not editiable

### ISSUE THREAD

**Issue by gawsoftpl** [tale/headplane#17]
Edit ACL tags is not editable

I see in code that edit acl tags has no action
```ts
<Menu.Item className='opacity-50 hover:bg-transparent'>
	Edit ACL tags
</Menu.Item>
```

I see that this project is great because looks like Real tailscale but there is some issue and features to deployment. Do you need a contributors?

**Comment by tale** [comment]
I haven't gotten around to it just yet. A PR is always appreciated!

**Comment by gawsoftpl** [comment]
Ok, I will create pull request when https://github.com/juanfont/headscale/pull/1792 this will be merged. I will create tags editor and acl editor in web ui

**Comment by tale** [comment]
Fixed in 7804d8318158b9d797072dda0f038a2679bb5e3f. Available in `0.2.1`.
